src/styles/blocks/card.scss

```
@@ -0,0 +1,114 @@
+.cards {
+  display: flex;
+  flex-wrap: wrap;
+  box-sizing: border-box;
+  margin: 0 auto;
+  //width: getContainerWidth($card-width, $card-gap, 40);
+  width: 1024px;
+  padding: 50px 40px;
+  
+  font-size: 10px;
+  line-height: 14px;
+  
+  &_centered {
+    margin: 0 auto;
+  }
+}
+
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid $border_color;
+  border-radius: 5px;
+  margin-right: 48px;
+  margin-bottom: 46px;
+
+  &:nth-child(4) {
+    margin-right: 0;
+  }
+
+  &:nth-child(8) {
+    margin-right: 0;
+  }
+
+  &_image {
+    margin: 0 auto 40px;
+    display: block;
+    height: 134px;
+    width: 160px;
+  }
+
+  &_product_name {
+    display: block;
+    margin: 0;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $primary_text_color;
+    margin-bottom: 4px;
+  }
+
+  &_product_code {
+    color: $default_text_color;
+    margin-bottom: 16px;
+  }
+
+  &_review {
+    display: flex;
+    align-items: flex-end;
+    justify-content: space-between;
+    margin: 0;
+    margin-bottom: 24px;
+
+    &_review_text {
+      color: $primary_text_color;
+    }
+  }
+
+  &_price {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    margin-bottom: 16px;
+    line-height: 18px;
+
+    &_text {
+      font-size: 12px;
+      color: $default_text_color;
+    }
+  
+    &_money {
+      font-size: 16px;
+      font-weight: 700;
+      line-height: 18px;
+      color: $primary_text_color;
+    }
+  }
+
+  .card_buy_button {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    box-sizing: border-box;
+    padding: 12px 0;
+    height: 40px;
+    width: 166px;
+    font-size: 14px;
+    line-height: 16px;
+    font-weight: 700;
+    text-transform: uppercase;
+    text-align: center;
+    text-decoration: none;
+    color: #fff;
+    background-color: $seondary_text_color;
```

Make sure there are no typos in your code
```suggestion
    background-color: $secondary_text_color;
```